typescript/src/ethereum.ts

```diff
@@ -244,4 +317,118 @@ export class Bridge implements ChainBridge {
       walletPublicKeyHash
     )
   }
+
+  // eslint-disable-next-line valid-jsdoc
+  /**
+   * @see {ChainBridge#deposits}
+   */
+  async deposits(
+    depositTxHash: TransactionHash,
+    depositOutputIndex: number
+  ): Promise<RevealedDeposit> {
+    const depositKey = this.buildDepositKey(depositTxHash, depositOutputIndex)
+
+    const deposit = await this._bridge.deposits(depositKey)
```

Comment:
Similar to above - shouldn't this be caleed `deposit` instead of `deposits`?

Reply:
See https://github.com/keep-network/tbtc-v2/pull/364#discussion_r895532994